airflow/providers/google/cloud/operators/life_sciences.py

```diff
@@ -90,5 +92,9 @@ def execute(self, context: 'Context') -> dict:
             api_version=self.api_version,
             impersonation_chain=self.impersonation_chain,
         )
-
+        LifeSciencesLink.persist(
+            context=context,
+            task_instance=self,
+            project_id=self.project_id or hook.project_id,
+        )
```

Comment:
to avoid problem with linter when you change the typing in the `persist` method, hide this link behind the `if`:
```python
project_id = self.project_id or hook.project_id
if project_id:
    LifeSciencesLink.persist(
        context=context,
        task_instance=self,
        project_id=project_id,
    )
```

Reply:
done